integrated_channels/canvas/migrations/0004_adding_learner_data_to_canvas.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.15 on 2020-08-31 23:13
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('canvas', '0003_delete_canvasglobalconfiguration'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CanvasLearnerDataTransmissionAudit',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('canvas_user_email', models.CharField(max_length=255)),
+                ('enterprise_course_enrollment_id', models.PositiveIntegerField(db_index=True)),
+                ('course_id', models.CharField(help_text="The course run's key which is used to uniquely identify the course for Degreed.", max_length=255)),
+                ('course_completed', models.BooleanField(default=True, help_text="The learner's course completion status transmitted to Degreed.")),
```

Comment:
haha Degreed!!

Reply:
NOOOOO 😒  I tried so hard to get all of them